.github/workflows/update-frozen.yml

```diff
@@ -46,6 +48,12 @@ jobs:
            tools/image-configure ${{ matrix.DEPLOYMENT_NAME }}  0
            df -h
 
+      # - name: Set Chilly Build
+      #   shell: bash
+      #   run: |
+      #     source setup-env
```

Comment:
Once you do "source setup-env" you should have the option to just "cd $IMAGE_DIR".

Reply:
removing this but it did NOT work for some reason